runtime/extensions/evm/src/executor/util.rs

```diff
@@ -0,0 +1,181 @@
+		Opcode(68) => "Difficulty",
+		Opcode(69) => "GasLimit",
+		Opcode(70) => "ChainId",
+		Opcode(80) => "Pop",
+		Opcode(81) => "MLoad",
+		Opcode(82) => "MStore",
+		Opcode(83) => "MStore8",
+		Opcode(84) => "SLoad",
+		Opcode(85) => "SStore",
+		Opcode(86) => "Jump",
+		Opcode(87) => "JumpI",
+		Opcode(88) => "GetPc",
+		Opcode(89) => "MSize",
+		Opcode(90) => "Gas",
+		Opcode(91) => "JumpDest",
+		Opcode(96) => "Push1",
+		Opcode(97) => "Push2",
+		Opcode(98) => "Push3",
+		Opcode(99) => "Push4",
+		Opcode(100) => "Push5",
+		Opcode(101) => "Push6",
+		Opcode(102) => "Push7",
+		Opcode(103) => "Push8",
+		Opcode(104) => "Push9",
+		Opcode(105) => "Push10",
+		Opcode(106) => "Push11",
+		Opcode(107) => "Push12",
+		Opcode(108) => "Push13",
+		Opcode(109) => "Push14",
+		Opcode(110) => "Push15",
+		Opcode(111) => "Push16",
+		Opcode(112) => "Push17",
+		Opcode(113) => "Push18",
+		Opcode(114) => "Push19",
+		Opcode(115) => "Push20",
+		Opcode(116) => "Push21",
+		Opcode(117) => "Push22",
+		Opcode(118) => "Push23",
+		Opcode(119) => "Push24",
+		Opcode(120) => "Push25",
+		Opcode(121) => "Push26",
+		Opcode(122) => "Push27",
+		Opcode(123) => "Push28",
+		Opcode(124) => "Push29",
+		Opcode(125) => "Push30",
+		Opcode(126) => "Push31",
+		Opcode(127) => "Push32",
+		Opcode(128) => "Dup1",
+		Opcode(129) => "Dup2",
+		Opcode(130) => "Dup3",
+		Opcode(131) => "Dup4",
+		Opcode(132) => "Dup5",
+		Opcode(133) => "Dup6",
+		Opcode(134) => "Dup7",
+		Opcode(135) => "Dup8",
+		Opcode(136) => "Dup9",
+		Opcode(137) => "Dup10",
+		Opcode(138) => "Dup11",
+		Opcode(139) => "Dup12",
+		Opcode(140) => "Dup13",
+		Opcode(141) => "Dup14",
+		Opcode(142) => "Dup15",
+		Opcode(143) => "Dup16",
+		Opcode(144) => "Swap1",
+		Opcode(145) => "Swap2",
+		Opcode(146) => "Swap3",
+		Opcode(147) => "Swap4",
+		Opcode(148) => "Swap5",
+		Opcode(149) => "Swap6",
+		Opcode(150) => "Swap7",
+		Opcode(151) => "Swap8",
+		Opcode(152) => "Swap9",
+		Opcode(153) => "Swap10",
+		Opcode(154) => "Swap11",
+		Opcode(155) => "Swap12",
+		Opcode(156) => "Swap13",
+		Opcode(157) => "Swap14",
+		Opcode(158) => "Swap15",
+		Opcode(159) => "Swap16",
+		Opcode(160) => "Log0",
+		Opcode(161) => "Log1",
+		Opcode(162) => "Log2",
+		Opcode(163) => "Log3",
+		Opcode(164) => "Log4",
+		Opcode(176) => "JumpTo",
+		Opcode(177) => "JumpIf",
+		Opcode(178) => "JumpSub",
+		Opcode(180) => "JumpSubv",
+		Opcode(181) => "BeginSub",
+		Opcode(182) => "BeginData",
+		Opcode(184) => "ReturnSub",
+		Opcode(185) => "PutLocal",
+		Opcode(186) => "GetLocal",
+		Opcode(225) => "SLoadBytes",
+		Opcode(226) => "SStoreBytes",
+		Opcode(227) => "SSize",
+		Opcode(240) => "Create",
+		Opcode(241) => "Call",
+		Opcode(242) => "CallCode",
+		Opcode(243) => "Return",
+		Opcode(244) => "DelegateCall",
+		Opcode(245) => "Create2",
+		Opcode(250) => "StaticCall",
+		Opcode(252) => "TxExecGas",
+		Opcode(253) => "Revert",
+		Opcode(254) => "Invalid",
+		Opcode(255) => "SelfDestruct",
+		_ => "Unknown", // TODO
```

Comment:
Are there any more pattern to match? Looks like you've covered all 255 already.

Reply:
Yeah, I was just thinking on some error handling when I coded this but is not possible to get an input value from the EVM that is not in the 0~255 range. We can remove it, `panic` or add an `unreachable` (I did the later).